python/cucim/setup.py

```diff
@@ -1,17 +1,13 @@
 #!/usr/bin/env python
 # -*- encoding: utf-8 -*-
-from __future__ import absolute_import
-from __future__ import print_function
+from __future__ import absolute_import, print_function
```

Comment:
We shouldn't need the `__future__` imports here since we are Python 3.x only

Reply:
The file is not used in RAPIDS CI/CD but will remove it.